pfb-analysis/scripts/export_connectivity.sh

```diff
@@ -58,36 +59,56 @@ function ec_export_table_shp() {
 
 function ec_export_table_csv() {
     OUTPUT_DIR="$1"
-    EXPORT_TABLENAME="$2"
+    OUTPUT_FILE_PREFIX="${2}"
+    EXPORT_TABLENAME="$3"
 
-    FILENAME="${OUTPUT_DIR}/${NB_OUTPUT_FILE_PREFIX}${EXPORT_TABLENAME}.csv"
+    FILENAME="${OUTPUT_DIR}/${OUTPUT_FILE_PREFIX}_${EXPORT_TABLENAME}.csv"
     psql -h "${NB_POSTGRESQL_HOST}" \
          -U "${NB_POSTGRESQL_USER}" \
          -p "${NB_POSTGRESQL_PORT}" \
          -d "${NB_POSTGRESQL_DB}" \
-         -c "COPY ${EXPORT_TABLENAME} TO '${FILENAME}' WITH (FORMAT CSV, HEADER)"
+         -c "\COPY ${EXPORT_TABLENAME} TO '${FILENAME}' WITH (FORMAT CSV, HEADER)"
 }
 
 if [ "${BASH_SOURCE[0]}" = "${0}" ]
 then
-    if [ "${1:-}" = "--help" ] || [ -z "${1:-}" ]
+    if [ "${1}" = "--help" ]
+    then
+        ec_usage
+    elif [ -z "${1}" ] || [ -z "${2}" ]
     then
         ec_usage
+        exit 1
     else
         echo "Exporting analysis to ${NB_OUTPUT_DIR}"
         NB_OUTPUT_DIR="${1}"
+        NB_OUTPUT_FILE_PREFIX="${2}"
         mkdir -p "${NB_OUTPUT_DIR}"
 
         # Export neighborhood_ways as SHP
-        ec_export_table_shp "${NB_OUTPUT_DIR}" "neighborhood_ways"
+        ec_export_table_shp "${NB_OUTPUT_DIR}" "${NB_OUTPUT_FILE_PREFIX}" "neighborhood_ways"
 
         # Export neighborhood ways as GeoJSON
         # TODO: Add ogr2ogr and try again
 
         # Export neighborhood_connected_census_blocks as SHP
-        ec_export_table_shp "${NB_OUTPUT_DIR}" "neighborhood_connected_census_blocks"
+        ec_export_table_shp "${NB_OUTPUT_DIR}" "${NB_OUTPUT_FILE_PREFIX}" "neighborhood_connected_census_blocks"
 
         # Export neighborhood_overall_scores as CSV
-        ec_export_table_csv "${NB_OUTPUT_DIR}" "neighborhood_overall_scores"
+        ec_export_table_csv "${NB_OUTPUT_DIR}" "${NB_OUTPUT_FILE_PREFIX}" "neighborhood_overall_scores"
+
+        if [ -v AWS_STORAGE_BUCKET_NAME ]
+        then
+          pushd "${NB_OUTPUT_DIR}"
+          sync  # Probably superfluous, but the s3 command said "file changed while reading" once
+
+          OUTPUT_FILENAME="${NB_OUTPUT_FILE_PREFIX}.zip"
+          zip "${OUTPUT_FILENAME}" "${NB_OUTPUT_FILE_PREFIX}_"*
+          sync
+
+          aws s3 cp "${OUTPUT_FILENAME}" "s3://${AWS_STORAGE_BUCKET_NAME}/results/"
```

Comment:
This is likely not going to work in the staging environment. There, we define a single storage bucket to hold all the analysis job results. We need to upload all files to a bucket scoped by the django job id (not currently provided to the analysis), otherwise files will end up clobbered. What if we uploaded our output files to:
- `s3://${AWS_STORAGE_BUCKET_NAME/results/${DJANGO_JOB_ID}/neighborhood_ways.zip`
- `s3://${AWS_STORAGE_BUCKET_NAME/results/${DJANGO_JOB_ID}/neighborhood_overall_scores.csv` 

That way, its trivial for the Django app to generate download links to the export files when a job is done.